apps/io-public/src/ValidateProfileEmail/index.ts

```diff
@@ -41,8 +40,8 @@ if (isLeft(errorOrValidationCallbackValidUrl)) {
 const validationCallbackValidUrl = errorOrValidationCallbackValidUrl.right;
 
 const tableClient = TableClient.fromConnectionString(
-  config.StorageConnection,
-  VALIDATION_TOKEN_TABLE_NAME
+  config.MAINTENANCE_STORAGE_ACCOUNT_CONNECTION_STRING,
```

Comment:
It looks like `StorageConnection` is no longer used, except in the health check. Could we remove it entirely and replace it with the new `MAINTENANCE_STORAGE_ACCOUNT_CONNECTION_STRING`?